src/main/java/nextstep/courses/service/ApproveService.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.courses.service;
+
+import nextstep.courses.domain.Session;
+import nextstep.courses.domain.Student;
+import nextstep.courses.infrastructure.JdbcStudentRepository;
+import nextstep.users.domain.NsUser;
+import nextstep.users.infrastructure.JdbcUserRepository;
+
+
+public class ApproveService {
+    private JdbcUserRepository userRepository;
+    private JdbcStudentRepository studentsRepository;
+    private SessionService sessionService;
+
+    public void approve(String userStringId, Long sessionId) {
```

Comment:
```suggestion
    public void approve(Long nsUserId, Long sessionId) {
```
NsUser의 id가 인자로 전달되지 않을까?